app/api/schema/image_sizes.py

```diff
@@ -35,3 +35,27 @@ class Meta:
     thumbnail_quality = fields.Integer(validate=lambda n: 0 <= n <= 100, allow_none=True)
     logo_width = fields.Integer(validate=lambda n: n >= 0, allow_none=True)
     logo_height = fields.Integer(validate=lambda n: n >= 0, allow_none=True)
+
+
+@use_defaults()
+class SpeakerImageSizeSchema(Schema):
+    """
+    Api schema for image_size Model
+    """
+    class Meta:
+        """
+        Meta class for image_size Api Schema
+        """
+        type_ = 'speaker-image-size'
+        self_view = 'v1.speaker-image_size_detail'
```

Comment:
Can we change this ? `speaker-image_size_detail` is weird. Both `-` and `_` shouldn't be used.